packages/component-library/src/storybook.css

```diff
@@ -0,0 +1,11 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
```

Comment:
Do we need to specify these again?

Reply:
Good call, I refactored the main storybook config file to import the `global.css` file and then import the `storybook.css` file (minus the @tailwind directives) to overwrite the attribute that was causing the issue. 

Originally wrote out `storybook.css` with the @tailwind directives and the overwrite to avoid any possible ui errors for the exported components that would be using `global.css` in that event that for some reason the overwrite did bork them